index.js

```diff
@@ -68,23 +88,55 @@ class HandlebarsRenderer {
     };
 
     /**
-     * Add templates to the active set of partials. The templates passed in have been
-     * run through the preProcessor function.
+     * Add templates to the active set of partials. The templates can either be raw
+     * template strings, or the result coming from the preProcessor function.
      *
-     * @param {Object} A set of pre-processed templates to add to the active set of partials
+     * @param {Object} A set of templates to register with handlebars
      */
     addTemplates(templates) {
-        _.each(templates, (precompiled, path) => {
-            // Take precompiled template and convert it into something that can
-            // be directly executed. Then, add it to the list of templates & partials.
-            if (typeof this.handlebars.partials[path] === 'undefined') {
-                let template;
-                eval(`template = ${precompiled};`); // evil, but necessary :/
-                this.handlebars.registerPartial(path, this.handlebars.template(template));
+        _.each(templates, (template, path) => {
```

Comment:
If the only reason `lodash` is included here is for looping, consider using the native `es6` implementations. It's probably microseconds savings, but also, maybe less `mb`s? 🍹

Reply:
We won't be able to get rid of lodash entirely because many of the helpers depend on it.